.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__/
 /build/
 /dist/
 /tmp/
+spdx/parsers/parsetab.py
```

Comment:
ok, so this is a problem with Ply. I cannot recall why I started  committing generated file but there was an issue AFAIRC, may be with wheels.